docs/instant.md

```diff
@@ -245,12 +249,16 @@ timestamp.toZonedDateTimeISO('UTC'); // => 2019-03-31T00:45+00:00[UTC]
 timestamp.toZonedDateTimeISO('-08:00'); // => 2019-03-30T16:45-08:00[-08:00]
 ```
 
-### instant.**toZonedDateTime**(_timeZone_: object | string, _calendar_: object | string) : Temporal.ZonedDateTime
+### instant.**toZonedDateTime**(_item_: object) : Temporal.ZonedDateTime
 
 **Parameters:**
 
-- `timeZone` (object or string): A `Temporal.TimeZone` object, or an object implementing the [time zone protocol](./timezone.md#protocol), or a string description of the time zone; either its IANA name or UTC offset.
-- `calendar` (object or string): A `Temporal.Calendar` object, or a plain object, or a calendar identifier.
+- `item` (object): an object with properties to be added to `time`. The following properties are
+  recognized:
+  - `calendar` (required `Temporal.Calendar` or value convertible to one, or an object implementing the calendar protocol):
```

Comment:
```suggestion
  - `calendar` (required calendar identifier string, `Temporal.Calendar` object, or object implementing the calendar protocol):
```

I found "value convertible to one" to be confusing because 99.9% of the time that value will be an ID string. I'd suggest we just say "string" here to be clearer.